Automattic/simplenote-electron
# Add Monospace Font Support to the Editor

This PR adds a new setting to allow display of a monospace font in the note editor.

Windows:
<img width="975" alt="screen shot 2018-01-02 at 2 53 46 pm" src="https://user-images.githubusercontent.com/789137/34503594-0ffd42fe-efce-11e7-850f-54177a0f1369.png">

Linux:
<img width="1257" alt="screen shot 2018-01-02 at 2 40 25 pm" src="https://user-images.githubusercontent.com/789137/34503598-166795f4-efce-11e7-8980-fff5888ddd29.png">

New menu item:
<img width="365" alt="screen shot 2018-01-02 at 2 44 34 pm" src="https://user-images.githubusercontent.com/789137/34503631-5a2c0324-efce-11e7-8b50-76528ef4da09.png">

**To Test**
 * Enable the monospace font, either in the 'View' menu or in the app settings dialog.
 * The editor should switch to the monospace font, and back to the default system font if disabled.

@drw158 'Courier' seemed to be the best looking cross-platform monospace font, but would love any additional input you may have.
